code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -3255,3 +3255,17 @@
 	catalysts = list(
 		/datum/reagent/enzyme = 1
 	)
+
+/datum/chemical_reaction/welderbomb
+	name = "Welder Bomb"
+	result = /datum/reagent/ethanol/welderbomb
+	required_reagents = list(/datum/reagent/ethanol/whiskey = 2, /datum/reagent/water = 1, /datum/reagent/fuel = 2)
```

Comment:
I don't quite get the purpose of this recipe, design-wise. 

If someone has access to whiskey, why would they bother cooking up a toxic recipe instead? Yes, it stretches out the supply, but not significantly so considering the toxicity.

The Rattle recipe is far more significant, because it provides an actual resource _source_.